mnist/mnist_ibm.ipynb

```diff
@@ -0,0 +1,948 @@
+{
```

Comment:
<p>(see Tip) - what does it point to?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/kubeflow/examples/pull/746/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Thanks, I added a new section pointing to the IBM COS.